azext_iot/_params.py

```diff
@@ -641,7 +641,11 @@ def load_arguments(self, _):
             options_list=["--auth-type"],
             arg_type=hub_auth_type_dataplane_param_type,
         )
-        context.argument("data", options_list=["--data", "--da"], help="Message body.")
+        context.argument(
```

Comment:
An advantage of a separate param/arg for reading from file / supporting binary is being able to iterate on it more easily in a _preview_ context.